File: Lecture 115/3.cpp
#include <iostream>
using namespace std;

class Parent {
public:
    int a;
};

class Child : public Parent {
public:
    int b;
};

int main() {
    Child obj;
    obj.a = 10;
    obj.b = 20;
    cout << obj.a << ", " << obj.b << endl;
    return 0;
}

File: Lecture 146/3.cs
using System;

class Parent {
    public double process(double value) {
        Console.Write("Parent process(double): ");
        return value + 2.5;
    }
}

class Child : Parent {
    public int process(int value) {
        Console.Write("Child process(int): ");
        return value + 3;
    }
}

class Program {
    static void Main() {
        Child obj = new Child();
        Console.WriteLine(obj.process(5));
        Console.WriteLine(obj.process(5.5));
    }
}
